terminfo/terminfo.go

```diff
@@ -616,27 +616,28 @@ func (t *Terminfo) TParm(s string, p ...int) string {
 	return pb.End()
 }
 
-// TPuts emits the string to the writer, but expands inline padding
-// indications (of the form $<[delay]> where [delay] is msec) to
-// a suitable number of padding characters (usually null bytes) based
-// upon the supplied baud.  At high baud rates, more padding characters
-// will be inserted.  All Terminfo based strings should be emitted using
-// this function.
-func (t *Terminfo) TPuts(w io.Writer, s string, baud int) {
+// InterpretDelays translates the input string by expanding inline padding
+// indications (of the form $<[delay]> where [delay] is msec) to a suitable
+// number of padding characters (usually null bytes) based upon the supplied
+// baud.  At high baud rates, more padding characters will be inserted. This
+// function is used by TPuts() to make strings suitable for sending to the
+// terminal.
+func (t *Terminfo) InterpretDelays(s string, baud int) string {
```

Comment:
I think this should be a private function.

Reply:
I'd actually call this "expandDelays".

Reply:
Hi - I just realised that the function in question is in the terminfo package but is called from the tcell package:

```go
res.WriteString(t.ti.ExpandDelays(codes, t.baud))
```
Do you feel strongly about not exporting this function from terminfo? 

Thanks

Reply:
Yes, but maybe we can be smarter about this somehow... I don't want this to be exposed to callers.

Reply:
Ok thanks, I will try to rework it.